testimonials.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Blue Velvet Church 🍔</title>
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    .🍔 {
+      background-image: url(./god.jpg);
+      background-position: center;
+      background-size: cover;
+      height: 100vh;
+      width: 100vw;
+      text-align: center;
+    }
+
+    .🗣️{
+    	float: left;
```

Comment:
Do not use float, please.

Otherwise it will curse you with McDonald's and BK.

Reply:
Sorry, the lack of Blue Velvet (about 2 weeks without one) made me do horrible stuff, like using float. I'm not in my best mental health conditions.

Praise the lord.